packages/@vue/cli-ui/apollo-server/util/notification.js

```diff
@@ -6,8 +6,16 @@ const builtinIcons = {
   error: path.resolve(__dirname, '../../src/assets/error.png')
 }
 
+// https://github.com/mikaelbr/node-notifier/issues/154
+// Specify appID to prevent SnoreToast shortcut installation.
+// SnoreToast actually uses it as the string in the notification's
+// title bar (different from title heading inside notification).
+// This only has an effect in Windows.
+const snoreToastOptions = notifier.Notification === notifier.WindowsToaster && { appID: 'Vue UI' }
```

Comment:
[`notifier` is set to `WindowsToaster`](https://github.com/mikaelbr/node-notifier/blob/29b99f2855f09ed3ee573e0410eb4ea4553e10f0/index.js#L30), which [uses SnoreToast](https://github.com/mikaelbr/node-notifier/blob/29b99f2855f09ed3ee573e0410eb4ea4553e10f0/notifiers/toaster.js#L5)


I only set `appID` for SnoreToast to avoid any effects on other notifiers that might also have an `appID` parameter.